exo-clone/src/service-cloner.ls

```diff
@@ -35,7 +36,7 @@ class ServiceCloner extends EventEmitter
 
 
   write: (text) ~>
-    @emit 'output', {@name, text: text.trim!.replace /\.*$/, ''}
+    @logger.log {@name, text: (text.trim!.replace /\.*$/, '')}
```

Comment:
Please use parentheses for normal function calls: 

```ls
text: text.trim!.replace(/\.*$/, '')
```

and not 

```
text: (text.trim!.replace /\.*$/, '')
```

We aren't writing LISP or Scheme here  :)